flink-end-to-end-tests/test-scripts/test_local_recovery_and_scheduling.sh

```diff
@@ -53,8 +53,6 @@ function cleanup_after_test {
     #
     kill ${watchdog_pid} 2> /dev/null
     wait ${watchdog_pid} 2> /dev/null
-    #
-    cleanup
```

Comment:
The `test_local_recovery_and_scheduling` test currently bundles several executions of the test (e.g. with different state backend configurations) in a single run of the test script. That's why it required this cleanup within the test itself.

How would the change of this PR affect this?
In general, should we also restructure e2e tests so that each execution configuration variant should be executed with the `test-runner-cleanup#run_test` method (instead of cleaning up itself in-between executions)?

AFAIK, only the `test_local_recovery_and_scheduling` does this at the moment.

Reply:
This should not be a concern anymore with the new changed where each configuration of `test_local_recovery_and_scheduling` is its own test-case